Examples/Tests/plasma_lens/inputs_test_3d_plasma_lens_python.py

```diff
@@ -0,0 +1,77 @@
+from pywarpx import (
+    Bucket,
+    Diagnostics,
+    Particles,
+    algo,
+    amr,
+    boundary,
+    diagnostics,
+    geometry,
+    particles,
+    picmi,
+    warpx,
+)
+
+clight = picmi.constants.c
+m_e = picmi.constants.m_e
+q_e = picmi.constants.q_e
+
+# Maximum number of time steps
+max_step = 84
+
+# number of grid points
+amr.n_cell = 16, 16, 16
+
+amr.max_level = 0
+
+# Geometry
+geometry.dims = "3"
```

Comment:
Because of the way this parameter is parsed, this must be a string.

Reply:
We could generalize `libwarpx` to assume Cartesian, if an integer is passed.